keras_cv/models/backbones/vit/vit_aliases.py

```diff
@@ -0,0 +1,401 @@
+            {
+                "include_rescaling": include_rescaling,
+                "input_shape": input_shape,
+                "input_tensor": input_tensor,
+            }
+        )
+        return ViTBackbone.from_preset("vits32", **kwargs)
+
+    @classproperty
+    def presets(cls):
+        """Dictionary of preset names and configurations."""
+        return {}
+
+    @classproperty
+    def presets_with_weights(cls):
+        """
+        Dictionary of preset names and configurations that include weights.
+        """
+        return {}
+
+
+class ViTB32Backbone(ViTBackbone):
+    def __new__(
+        cls,
+        include_rescaling=True,
+        input_shape=(224, 224, 3),
+        input_tensor=None,
+        **kwargs
+    ):
+        kwargs.update(
+            {
+                "include_rescaling": include_rescaling,
+                "input_shape": input_shape,
+                "input_tensor": input_tensor,
+            }
+        )
+        return ViTBackbone.from_preset("vitb32", **kwargs)
+
+    @classproperty
+    def presets(cls):
+        """Dictionary of preset names and configurations."""
+        return {}
+
+    @classproperty
+    def presets_with_weights(cls):
+        """
+        Dictionary of preset names and configurations that include weights.
+        """
+        return {}
+
+
+class ViTL32Backbone(ViTBackbone):
+    def __new__(
+        cls,
+        include_rescaling=True,
+        input_shape=(224, 224, 3),
+        input_tensor=None,
+        **kwargs
+    ):
+        kwargs.update(
+            {
+                "include_rescaling": include_rescaling,
+                "input_shape": input_shape,
+                "input_tensor": input_tensor,
+            }
+        )
+        return ViTBackbone.from_preset("vitl32", **kwargs)
+
+    @classproperty
+    def presets(cls):
+        """Dictionary of preset names and configurations."""
+        return {}
+
+    @classproperty
+    def presets_with_weights(cls):
+        """
+        Dictionary of preset names and configurations that include weights.
+        """
+        return {}
+
+
+class ViTH32Backbone(ViTBackbone):
+    def __new__(
+        cls,
+        include_rescaling=True,
+        input_shape=(224, 224, 3),
+        input_tensor=None,
+        **kwargs
+    ):
+        kwargs.update(
+            {
+                "include_rescaling": include_rescaling,
+                "input_shape": input_shape,
+                "input_tensor": input_tensor,
+            }
+        )
+        return ViTBackbone.from_preset("vith32", **kwargs)
+
+    @classproperty
+    def presets(cls):
+        """Dictionary of preset names and configurations."""
+        return {}
+
+    @classproperty
+    def presets_with_weights(cls):
+        """
+        Dictionary of preset names and configurations that include weights.
+        """
+        return {}
+
+
+ALIAS_DOCSTRING = """ViTBackbone Model with the {name} configuration for {patch_size} patch size.
```

Comment:
We should probably mention the 224x224 default size in this docstring

Reply:
Yeah, done!